### PR TITLE
docs: Update lzv2 README

### PR DESCRIPTION
**Pull request by cartyc**
Added details on package structure and new diagram and added current pkg version to kpt commands

### CODE REVIEW

File: docs/landing-zone-v2/README.md [hunk 1]
```diff
@@ -59,17 +60,32 @@ This Landing Zone v2 assumes that the different required environments known as E
...
 
+A deployed Landing Zone contains two sections the Core Landing Zone and the Client Landing Zone. Together they will deploy a structure that matches the diagram below.
+
+
 ![img](img/folder-structure.png)
 
-### Core Landing Zone Folders
+### Core Landing Zone
+
+This is made up of two pacakges `core-landing-zone` and `gatekeeper-policies`. This section provides the core of the landing zone functionality with Logging, Org Policies, and Shared Infrastructure like DNS and a Shared VPC (coming soon) as well as compliance enforcement in Config Controller.
```
Comment (davelanglois-ssc) [inline docs/landing-zone-v2/README.md:86]: typo packages

**Review by johnswayty-ssc — APPROVED** [review]

**Review by davelanglois-ssc — COMMENTED** [review]
is there a major reason why you want to embed the package versions in the doc ?

File: docs/landing-zone-v2/README.md (hunk 1 above)
Comment (cartyc) [reply docs/landing-zone-v2/README.md:86]: @davelanglois-ssc to smooth out the onboarding experience. We've gone through a few setups recently with clients and this is a stumbling block and slows things down. It's a bit more work on our end to keep the docs up and running but lets users simply run the command without having to bounce between tabs to find the correct versions.

**Review by fmichaelobrien — COMMENTED** [review]
comma below
two sections, the...

**Review by fmichaelobrien — COMMENTED** [review]
for core-landing-zone and client lz packages - we may want to hyperlink to the packages - to avoid ambiguity

**Review by fmichaelobrien — COMMENTED** [review]
for the deployment set 0.2.0, 0.1.0 and 0.3.2 we may want to document on a line this release set - and that it is a current example - to be modified from <version> in the future

**Review by fmichaelobrien — DISMISSED** [review]
LGTM - after the hardcoded edits/explanations and minor edits

**Review by fmichaelobrien — APPROVED** [review]
LGTM
the readme community will appreciate this

**Comment by cartyc** [comment]
@davelanglois-ssc added a disclaimer on the package versions. I think keeping the current versions in the commands keeps things easier for now until we can add in a command to grab the correct versions for automation.

**Review by fmichaelobrien — APPROVED** [review]
LGTM
Some lint rules are better than others

File: docs/landing-zone-v2/README.md [hunk 2]
```diff
@@ -396,25 +396,33 @@ cd pbmm-landingzone
 
 1. Get the gatekeeper policies package. These are the policies which your Config Controller instance will use to [validate](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller) compliance of any configuration deployed, this currently checks for guardrails compliance and naming conventions on projects. These policies will also be validated when running the `kpt fn render` command for early feedback.
 
+   The latest versions of the releases can be found in [Releases](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/releases). The provided versions are examples and may not be up to date.
```
Comment (davelanglois-ssc) [inline docs/landing-zone-v2/README.md:399]: here is the code snippet you can use to extract the latest version
URL="https://raw.githubusercontent.com/GoogleCloudPlatform/pubsec-declarative-toolkit/main/.release-please-manifest.json"
PACKAGE="solutions/client-setup"

VERSION=$(curl -s $URL | jq -r ".\"$PACKAGE\"")

**Review by lucstjean-ssc — DISMISSED** [review]
/lgtm

File: docs/landing-zone-v2/README.md (hunk 2 above)
Comment (davelanglois-ssc) [reply docs/landing-zone-v2/README.md:399]: @cartyc let me know if you are considering implement the suggested code now or if you prefer to do it later.

Comment (cartyc) [reply docs/landing-zone-v2/README.md:399]: Added the changes yesterday and didn't have time to push the code. Incoming :)

**Review by fmichaelobrien — APPROVED** [review]
LGTM

**Pull request merged**